.github/workflows/commit_epm.yml

```diff
@@ -29,10 +29,10 @@ jobs:
     - name: Build install packages and run epm service
       run: |
         docker exec $rune_test bash -c "mkdir -p /var/run/epm;
-        mkdir -p /var/epm;
+        mkdir -p /etc/epm;
         mkdir -p /var/local/epm;
```

Comment:
Maybe it is no need to `mkdir /var/local/epm`?
Since:
/etc/epm/config.toml 
/etc/epm/epm.db
/var/run/epm/epm.sock

Reply:
/var/local/epm is for cache bundle data.